doc/howto/sms_reminder_experiments.md

```diff
@@ -0,0 +1,78 @@
+# Running SMS reminder experiments
+
+## Prerequisites
+To start an SMS experiment you'll need to gather the following information:
+
+- A `name` for the experiment.
+- The `type` of experiment - it can target either "Current" patients or "Stale" patients.
+- The number of `max_patients_per_day` that will be enrolled per day.
+- `start_time` - enrollments will begin at this time.
+- `end_time` - enrollments will stop at this time .
+- `treatment_groups` - Each treatment group will have a different content or frequency of messages sent to it.
+    - You will probably also need a `control` group as a baseline in addition to all the groups that will receive messages.
+- The content and frequency of messages in each treatment group. This will be captured in the
+  `reminder_templates` of each group.
+  For current patients, the `remind_on` of reminder templates is set relative to the date
+  they are expected to return. For stale patients, it's the date of enrollment.
```

Comment:
This bullet is still a bit opaque to new folks. Can we elaborate in great detail with some examples? It would be very helpful to show how to set up a 3-message cascade for current patients for, say, -1, 0, and +3 days from appt.